src/assets/scss/reset.scss

```diff
@@ -290,7 +290,7 @@ select {
  */
 button,
 html [type='button'],
-/* 1 */
+  /* 1 */
```

Comment:
What's this about?

Reply:
The old extra spacing that I didn't intend issue.